index.src.html

```diff
@@ -1550,33 +1550,41 @@ <h3 id="violation-events">
     [Constructor(DOMString type, optional SecurityPolicyViolationEventInit eventInitDict),
      Exposed=(Window,Worker)]
     interface SecurityPolicyViolationEvent : Event {
-        readonly    attribute USVString      documentURI;
+        readonly    attribute USVString      documentURL;
+        readonly    attribute USVString      documentURI;         // historical
         readonly    attribute USVString      referrer;
-        readonly    attribute USVString      blockedURI;
+        readonly    attribute USVString      blockedURL;
+        readonly    attribute USVString      blockedURI;          // historical
         readonly    attribute DOMString      violatedDirective;
         readonly    attribute DOMString      effectiveDirective;
         readonly    attribute DOMString      originalPolicy;
         readonly    attribute USVString      sourceFile;
         readonly    attribute DOMString      sample;
         readonly    attribute SecurityPolicyViolationEventDisposition      disposition;
         readonly    attribute unsigned short statusCode;
-        readonly    attribute unsigned long  lineNumber;
-        readonly    attribute unsigned long  columnNumber;
+        readonly    attribute unsigned long  lineno;
+        readonly    attribute unsigned long  lineNumber;          // historical
+        readonly    attribute unsigned long  colno;
+        readonly    attribute unsigned long  columnNumber;        // historical
     };
 
     dictionary SecurityPolicyViolationEventInit : EventInit {
-        required USVString      documentURI;
+        required USVString      documentURL;
+        required USVString      documentURI;                      // historical
                  USVString      referrer = "";
-                 USVString      blockedURI = "";
+                 USVString      blockedURL = "";
+                 USVString      blockedURI = "";                  // historical
         required DOMString      violatedDirective;
         required DOMString      effectiveDirective;
         required DOMString      originalPolicy;
                  USVString      sourceFile = "";
                  DOMString      sample = "";
         required SecurityPolicyViolationEventDisposition disposition;
         required unsigned short statusCode;
-                 unsigned long  lineNumber = 0;
-                 unsigned long  columnNumber = 0;
+                 unsigned long  lineno = 0;
+                 unsigned long  lineNumber = 0;                   // historical
+                 unsigned long  colno = 0;
+                 unsigned long  columnNumber = 0;                 // historical
     };
   </pre>
 
```

Comment:
Perhaps add something like:

```suggestion
The <dfn for="SecurityPolicyViolationEvent" attribute export>lineNumber</dfn> attribute's getter returns the value of the object's {{SecurityPolicyViolationEvent/lineno}} property.

The <dfn for="SecurityPolicyViolationEvent" attribute export>columnNumber</dfn> attribute's getter returns the value of the object's {{SecurityPolicyViolationEvent/colno}} property.
```

Reply:
Done.

Reply:
I think I understand better the feedback now.

I've defined the old attribute's getters as retrieving the values of the new attributes.

I also took the liberty of doing the same thing for violated-directive as it's a copy of effective-directive.